oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -342,6 +342,10 @@ receiver-async-profiler:
   default:
     # Used to manage the maximum size of the jfr file that can be received, the unit is Byte, default is 30M
     jfrMaxSize: ${SW_RECEIVER_ASYNC_PROFILER_JFR_MAX_SIZE:31457280}
+    # Used to decide whether to use createTempFile or ByteBuf to receive JFR files.
+    # If true, createTempFile is used to receive JFR files, otherwise ByteBuf is used to receive JFR files.
+    # The latter is used by default. For details, please refer to the comments of class AsyncProfilerModuleConfig
+    tempFileReceiverEnabled: ${SW_RECEIVER_ASYNC_PROFILER_TEMP_FILE_RECEIVER_ENABLED:false}
```

Comment:
You could rename this as `memoryParserEnabled`.

And explanation should be about `memory file` and `physical file` modes. How we wrote codes doesn't matter for end users.
We should explain the pros and cons. Memory files have fewer local file system limitations, so they are by default. But it costs more memory. When users need to parse large JFR files(rare case), they could switch to file mode.